### PR TITLE
242-valid-anagram

### LINKED ISSUE #3: 242. Valid Anagram
Given two strings 's' and 't', return 'true' if ''t is an anagram of 's', and 'false' otherwise.

An **Anagram** is a word or phrase formed by rearranging the letters of a different word or phrase, typically using all the original letters exactly once.

**Example 1:**

Input: s = "anagram", t = "nagaram"
Output: true

**Example 2:**

Input: s = "rat", t = "car"
Output: false
 

**Constraints:**

'1 <= s.length, t.length <= 5 * 104'
's' and 't' consist of lowercase English letters.

**Code example:**
```
public class Solution {
    public bool IsAnagram(string s, string t) {
        
    }
}
```

### PR DESCRIPTION
**Pull request by LoLProM**

### CODE REVIEW

File: 242.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    public bool IsAnagram (string s, string t){
+        if (s.Length != t.Length) return false;
+        char[] lettersOf_S = s.ToCharArray();
+        Array.Sort(lettersOf_S);
+        char[] lettersOf_T = t.ToCharArray();
+        Array.Sort(lettersOf_T);
+        return IsAnagramVerified(lettersOf_S,lettersOf_T);
+    }
+
+    private bool IsAnagramVerified(char[] s, char[] t, int i = 0, int j = 0){
```
Comment (andyguerra) [inline 242.cs]: There is no need to make this method recursive, that will degrade the performance of the algorithm and doesn't make the code more readable

**Review by andyguerra — CHANGES_REQUESTED** [review]

File: 242.cs (hunk 1 above)
Comment (LoLProM) [reply 242.cs]: tabien

**Comment by andyguerra** [comment]
The first solution was Success:
Runtime: 87 ms, faster than 51.53% of C# online submissions for Valid Anagram.
Memory Usage: 39.8 MB, less than 75.86% of C# online submissions for Valid Anagram.

Voy a ver la segunda

**Comment by andyguerra** [comment]
Please don't keep both solutions in the same file, fix the second solution so that the file contains only the second solution.

**Comment by LoLProM** [comment]
ya la subi aparte

File: .DS_Store
Comment (andyguerra) [inline .DS_Store:1]: Delete this file, it is not needed in the repo

File: 242.cs
Comment (andyguerra) [inline 242.cs:1]: **Wrong Answer:**
`Input "aacc" "ccac"`
`Output true`
`Expected false`

I get the idea, try to analyze why that doesn't works. You are close. Try to solve it. 

Check this [video](https://www.youtube.com/watch?v=9UtInBqnCgA) even you you solve it by yourself

**Review by andyguerra — CHANGES_REQUESTED** [review]

**Review by andyguerra — APPROVED** [review]
Aprobado, recuerda ver el vídeo

**Pull request merged**